### PR TITLE
Read user's display name directly from database!

### PR DESCRIPTION
**Pull request by DFurnes**
### What's this PR do?

This pull request replaces the `getUserById` GraphQL query with directly reading from the related model through Eloquent! This takes advantage of [jenssegers/laravel-mongodb](https://github.com/jenssegers/laravel-mongodb)'s [cross-database relationship](https://github.com/jenssegers/laravel-mongodb#cross-database-relationships) support.

I also added some missing model imports in factory classes.

### How should this be reviewed?

It may be easiest to review this commit-by-commit.

🏭 I added missing `App\Models` imports to factories in ac8137f.

👨‍👩‍👧‍👦 I updated our seeders to create real MongoDB users for activity (rather than [seeded IDs](https://github.com/DoSomething/northstar/blob/37a13e7969e4a46ba43bfa4a5182d0f2f2fdcfcf/database/faker/FakerNorthstarId.php)) in a41c794. This gives proper relationships between users & their activity on local.

🥞 I directly reads the user model from Eloquent, rather than GraphQL, in 624d610.

🏷️ Finally, I configured model relationships (e.g. `$signup->user`) in 1f289e2.

### Any background context you want to provide?

🥞

### Relevant tickets

References [Pivotal #176469276](https://www.pivotaltracker.com/story/show/176469276).

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added appropriate feature/unit tests.
- [ ] If new attributes were added to users, there is a corresponding PR to surface these in Aurora.
- [ ] If new attributes were added to users, then the data team already knows about these changes.

### CODE REVIEW

File: app/Console/Commands/SetupCommand.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 namespace App\Console\Commands;
 
+use Defuse\Crypto\Key;
```
Comment (weerd) [inline app/Console/Commands/SetupCommand.php:5]: Did this somehow get removed again? Or did we never put it back?

**Review by weerd — APPROVED** [review]
🙌  🎉

File: app/Console/Commands/SetupCommand.php (hunk 1 above)
Comment (DFurnes) [reply app/Console/Commands/SetupCommand.php:5]: Whoah, that's spooky. I swear we added that the other day (and I definitely didn't intentionally add it today)...

Comment (weerd) [reply app/Console/Commands/SetupCommand.php:5]: 👻

**Pull request merged**